democracy/views/contact_person.py

```diff
@@ -1,10 +1,18 @@
 from rest_framework import mixins, permissions, response, serializers, status, viewsets
+from rest_framework.permissions import IsAuthenticated
 
 from democracy.models import ContactPerson, Organization
 from democracy.pagination import DefaultLimitPagination
 from democracy.views.utils import TranslatableSerializer
 
 
+class ContactPersonPermission(IsAuthenticated):
+    message = "User without organization cannot access contact persons."
+
+    def has_permission(self, request, view):
+        return bool(super().has_permission(request, view) and request.user.get_default_organization())
```

Comment:
If you make custom permissions, inherit them from `BasePermission`. Inheriting from anything else can obfuscate the code and can act as a potential source of spaghetti code.

I assume the authenticatio part doesn't really matter for this permission either; what matters is that the method exists. So could do it something like this instead:
```suggestion
class ContactPersonPermission(permissions.BasePermission):
    message = "User without organization cannot access contact persons."

    def has_permission(self, request, view):
        return bool(hasattr(request.user, "get_default_organization") and request.user.get_default_organization())
```